public/locales/cn/auth.json

```diff
@@ -21,5 +21,18 @@
       "description": "没注册吗？",
       "action": "创建一个吧!"
     }
+  },
+  "forget-password": {
+    "title": "忘记密码",
+    "page-title": "告诉我们你的邮箱吧！",
+    "page-subtitle": "我们会一个重置密码链接到你的邮箱里",
+    "email": {
+      "label": "你的邮箱",
+      "placeholder": "邮箱",
+      "schema": "请输入你的邮箱"
```

Comment:
Imagine there are multiple validation rules, then this is better to be
```
{
  schema: {
    required,
    other-rules 
  }
}

Reply:
this has been updated